packages/apify/src/actor.ts

```diff
@@ -107,9 +106,9 @@ export function getEnv(): ApifyEnv {
         if (value && fullName.endsWith('_AT')) value = tryParseDate(value);
         else if (_.contains(INTEGER_ENV_VARS, fullName)) value = parseInt(value!, 10);
 
-        envVars[camelCaseName] = value || value === 0
+        Reflect.set(envVars, camelCaseName, value || value === 0
```

Comment:
Typescript really hates setting the value to null for some reason, even if the types say that `null` can be one of the possible values, which beats me..
```
Type 'string | number | Date | null' is not assignable to type 'null'.
  Type 'string' is not assignable to type 'null'.ts(2322)
```

Reply:
you cant assign `A | B` to `A` as it can have the type of `B` and that is not assignable. it should work with intersection (which is obviously not applicable here), with union it works the other way.